docs/connector-development/connector-builder-ui/partitioning.md

```diff
@@ -0,0 +1,116 @@
+* The [Woocommerce API](https://woocommerce.github.io/woocommerce-rest-api-docs/#order-notes) includes an endpoint to fetch notes of webshop orders via the `/orders/<id>/notes` endpoint. The `<id>` placeholder needs to be set to the id of the order to fetch the notes for. The orders represent the partitions of the notes stream.
+
+There are some cases that require multiple requests to fetch all records as well, but partitioning is not the right tool to configure these in the connector builder:
+* If your records are spread out across multiple pages that need to be requested individually if there are too many records, use the Pagination feature.
+* If your records are spread out over time and multiple requests are necessary to fetch all data (for example one request per day), use the Incremental sync feature.
+
+## Dynamic and static partition routing
+
+There are three possible sources for the partitions that need to be queried - the connector itself, supplied by the end user when configuring a Source based on the connector, or the API  provides the list of partitions on another endpoint (for example the Woocommerce API also includes an `/orders` endpoint that returns all orders).
+
+The first two options are a "static" form of partition routing (because the partitions won't change as long as the Airbyte configuration isn't changed). The API providing the partitions via one or multiple separate requests is a "dynamic" form of partition routing because the partitions can change any time.
+
+### List partition router
+
+To configure static partitioning, choose the "List" method for the partition router. The following fields have to be configured:
+* The "partition values" can either be set to a list of strings, making the partitions part of the connector itself or delegated to a user input so the end user configuring a Source based on the connector can control which partitions to fetch.
+* The "Current partition value identifier" can be freely choosen and is the identifier of the variable holding the current partition value. It can for example be used in the path of the stream using the `{{ stream_slice.<identifier> }}` syntax.
+* The "Inject partition value into outgoing HTTP request" option allows to configure how to add the current partition value to the requests
+
+#### Example
+
+To enable static partition routing for the [SurveySparrow API](https://developers.surveysparrow.com/rest-apis/response#getV3Responses) responses, the list partition router needs to be configured as following:
+* "Partition values" are set to the list of survey ids to fetch
+* "Current partition value identifier" is set to `survey` (this is not used for this example)
```

Comment:
I don't understand "(this is not used for this example)". Sounds like it's an implementation detail - can we remove it?